bedrock/firefox/views.py

```diff
@@ -557,6 +565,37 @@ def get_template_names(self):
         return [template]
 
 
+WNP117_VPN_EXPANSION_COUNTRIES = [
+    "AT",  # Austria
+    "BE",  # Belgium
+    "BG",  # Bulgaria
+    "CH",  # Switzerland
+    "CY",  # Cyprus
+    "CZ",  # Czech Republic
+    # "DE",  # Germany
```

Comment:
Nit: remove unused country codes?

Reply:
Confirmed with Caro that France and Germany are NOT to be targeted so I'll remove these (I had commented them out until I could confirm, thanks for reminding me).